spec/label_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'date'
+require_relative '../classes/label'
+
+describe Label do
+  context 'When testing the Label class' do
+    it 'The add_items method should add item to @@items' do
+      label = Label.new('Whatever', 'red')
+      item1 = Item.new(Date.today.to_s, true)
+
+      label.add_item(item1)
+      items = Label.class_variable_get(:@@items)
+      expect(items).to include(item1)
+    end
+  end
+end
```

Comment:
- [ ] Good job adding this test. However, this test is failing, let's try to fix it.

<img width="618" alt="chrome_lDZ1YGxG59" src="https://user-images.githubusercontent.com/81468675/154713343-c5ff8a82-8b52-4fa6-b186-14a000acf1d4.png">